R/mmrm-methods.R

```diff
@@ -192,6 +193,7 @@ print.summary.mmrm <- function(x,
   cat("mmrm fit\n\n")
   h_print_call(x$call, x$n_obs, x$n_subjects, x$n_timepoints)
   h_print_cov(x$cov_type, x$n_theta, x$n_groups)
+  cat("Adjust:      ", x$method, "\n", sep = "")
   cat("Method:      ")
```

Comment:
hm so maybe we should replace `Method` here by `Inference` ? and use `Method` instead of `Adjust` above?